server/config/playground.go

```diff
@@ -0,0 +1,25 @@
+package config
+
+import "time"
+
+type PlaygroundConfig struct {
+	Port                       int           `default:"8080"`
+	Debug                      bool          `default:"false"`
+	AllowedOrigins             []string      `default:"http://localhost:3000"`
+	SessionAuthKey             string        `default:"428ce08c21b93e5f0eca24fbeb0c7673"`
+	SessionMaxAge              time.Duration `default:"157680000s"`
+	SessionCookiesSecure       bool          `default:"true"`
+	SessionCookiesHTTPOnly     bool          `default:"true"`
+	SessionCookiesSameSiteNone bool          `default:"false"`
+	LedgerCacheSize            int           `default:"128"`
+	PlaygroundBaseURL          string        `default:"http://localhost:3000"`
+	StorageBackend             string
+	ForceMigration             bool `default:"false"`
```

Comment:
this is not linted, it should be aligned

Reply:
The linter disagrees since it's on a line below the other aligned ones. I can move it up if it's painful for you 👀